Jenkinsfile

```diff
@@ -274,23 +284,22 @@ pipeline {
 
     environment {
         GITHUB_USER = credentials('daos-jenkins-review-posting')
-        BAHTTPS_PROXY = "${env.HTTP_PROXY ? '--build-arg HTTP_PROXY="' + env.HTTP_PROXY + '" --build-arg http_proxy="' + env.HTTP_PROXY + '"' : ''}"
-        BAHTTP_PROXY = "${env.HTTP_PROXY ? '--build-arg HTTPS_PROXY="' + env.HTTPS_PROXY + '" --build-arg https_proxy="' + env.HTTPS_PROXY + '"' : ''}"
-        UID = sh script: "id -u", returnStdout: true
-        BUILDARGS = "$env.BAHTTP_PROXY $env.BAHTTPS_PROXY "                   +
-                    "--build-arg NOBUILD=1 --build-arg UID=$env.UID "         +
-                    "--build-arg JENKINS_URL=$env.JENKINS_URL "               +
-                    "--build-arg CACHEBUST=${currentBuild.startTimeInMillis}"
         SSH_KEY_ARGS = "-ici_key"
         CLUSH_ARGS = "-o$SSH_KEY_ARGS"
-        QUICKBUILD_DEPS = sh script: "rpmspec -q --srpm --requires utils/rpms/daos.spec 2>/dev/null",
+        UID = getuid()
+        BUILDARGS = docker_build_args()
+        BUILDARGS_QB_CHECK = docker_build_args(qb: quickbuild)
+        BUILDARGS_QB_TRUE = docker_build_args(qb: true)
+        QUICKBUILD_DEPS = sh script: 'rpmspec -q --srpm --requires' +
+                                     ' utils/rpms/daos.spec 2>/dev/null',
                              returnStdout: true
         TEST_RPMS = cachedCommitPragma(pragma: 'RPM-test', def_val: 'false')
     }
 
     options {
         // preserve stashes so that jobs can be started at the test stage
         preserveStashes(buildCount: 5)
+        ansiColor('xterm')
```

Comment:
How/where are we using this?

Reply:
I saw escape sequences in the logs.  This will render them in the log, and probably make them more obvious.
I do not know which components are generating them.